config-template.json

```diff
@@ -105,7 +104,7 @@
             {
               "id": "16A19515-C2B9-4982-80BD-69BED6A86C85",
               "name": "basic-public",
-              "description": "Provides a single publicly accessible S3 bucket with unlimited storage.  All data in this bucket is public.",
+              "description": "A single bucket with unlimited storage where the files are all public to read",
```

Comment:
Suggest "A single public bucket with unlimited storage, where the files are all public to read" to match the phrasing above ("A single private bucket with unlimited storage"). It's ok that it's a bit redundant, since it's really important for customers to know that it's public.